### ISSUE TITLE
Add support for special characters in DATABASE_URL

### ISSUE THREAD

**Issue by robjuz** [kimai/kimai#4975]
### Describe the issue

Technically its supported by symfony https://symfony.com/doc/6.4/doctrine.html#configuring-the-database . The problem is the `testdb.php` script.

## Solution:

add `urldecode` for the params
`dbtest.php`
```php
<?php
$DB_HOST = urldecode($argv[1]);
$DB_BASE = urldecode($argv[2]);
$DB_PORT = $argv[3];
$DB_USER = urldecode($argv[4]);
$DB_PASS = urldecode($argv[5]);

```

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.18.0

### How do you run Kimai?

Docker

### Which PHP version are you using?

8.1

**Comment by kevinpapst** [comment]
Up for a PR?

**Comment by kevinpapst** [comment]
Fixed by #4976
